site/source/docs/embark_commands.md

```diff
@@ -151,6 +151,16 @@ $ embark reset
 
 Resets embarks state on this dapp including clearing cache.
 
+## exec
+
+```
+$ embark exec [environment] [file|directory]
+```
+
+Executes a given (migration) script to perform complex after deployment operations.
+
+It's required to specifiy the `environment` in which the script(s) will be executed in. In addition it's possible to specificy a directory in which multiple script live in. Embark will execute them one by one.
```

Comment:
> It's required to specify the `environment`

Doesn't it default to `development`?

Reply:
Yes it does, but based on the spec, the command is:

```
$ embark exec [environment] [file|dir]
```

^ positional arguments.